src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -61,18 +62,18 @@ public String getCaseId() {
     }
 
     public AppealHearingType getHearingType() {
-        String hearingType = responseWrapper.getNewSscsCaseData().getAppeal().getHearingType();
+        final String hearingType = responseWrapper.getNewSscsCaseData().getAppeal().getHearingType();
+        AppealHearingType returnHearingType = REGULAR;
         if (HEARING_TYPE_ONLINE_RESOLUTION.equalsIgnoreCase(hearingType)) {
```

Comment:
This should be a series of if...then...else statements and not a bunch of if statements which will all be evaluated.